src/importexport/midi/internal/midishared/midifile.cpp

```diff
@@ -552,7 +552,7 @@ bool MidiFile::readEvent(MidiEvent* event)
         }
     }
 
-    unsigned char* data;
+    std::vector<unsigned char> data;
```

Comment:
I believe it's officially not entirely correct to reuse a vector after `std::move`ing it. Instead, you should probably make this variable as local as possible. Basically replace the `resize` calls with `std::vector<unsigned char> data(len + 1);`.